Cargo.toml

```diff
@@ -37,6 +37,7 @@ members = [
     "rmdir",
     "seq",
     "sleep",
+    "sort",
```

Comment:
Can you add it to the other toml files so it can be tested on all possible platforms?